### ISSUE TITLE
Wrong types for ButtonGroup component

### ISSUE THREAD

**Issue by vdimitrovv** [react-native-training/react-native-elements#1852]
## Explain what you did
I was trying to add a couple of options to a form by using `<ButtonGroup />`. I wanted to not have a default selected index in order to force the user to tap on the desired option, but the `selectedIndex` prop only allows values of type `number` to be used.

## Expected behaviour
If `selectedIndex` is `null` then don't highlight any options (behaviour already there, but types are wrong).

## Steps to reproduce 
1. Create an empty view with `<ButtonGroup />` inside it.
3. Set `selectedIndex` to `null` and enjoy the warnings/errors depending on whether or not you're using TypeScript.

## Possible Solution
```
/**
   * Current selected index of array of buttons
   */
  selectedIndex: number | null;
```

## Environment

| software              | version |
| --------------------- | ------- |
| react-native-elements |     1.1.0    |
| react-native          |    0.58.3     |
| npm or yarn           |     1.13.0    |

**Comment by hshbaklo** [comment]
same error when trying to use `selectMultiple` option. the type is then an array of numbers but current prop-type only accept a number.

**Comment by iRoachie** [comment]
Fixed in https://github.com/react-native-training/react-native-elements/pull/1994. Will be available in 1.2.0.

It's also optional now in typescript so you don't need to set it and it'll default to null.
